### PR TITLE
Add PDF docs with release

### LINKED ISSUE #23: Generate PDF docs with release
### Description of the feature

Would be great to add pdf generation with this release.

### Steps for implementing the feature

See https://github.com/pyansys/dev-guide/blob/af838bdde0e5e8e5451e348d8f1c9a243448d7b1/.github/workflows/ci-build.yml#L55-L74

@Revathyvenugopal162 is a great resource if you need help implementing it.

### Useful links and references

See [Building PDF Documentation](https://dev.docs.pyansys.com/how-to/documenting.html#building-pdf-documentation)

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
This PR will fix #23 to generate PDF Files with release.
 
- [x] Add LaTex elements to conf file.
- [x] Add workflow. 
- [x] Update Makefile.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -173,13 +173,27 @@ jobs:
         run: |
           make -C doc html
 
+      - name: Build PDF Documentation
+        run: |
+          sudo apt update
+          sudo apt-get install -y texlive-latex-extra latexmk
+          pip install .[doc]
```
Comment (RobPasMue) [inline .github/workflows/ci.yml:180]:
```suggestion
```
You do not need to install the pip reqs again. They have already been installed in the previous step

File: doc/make.bat [hunk 2]
```diff
@@ -37,5 +38,11 @@ goto end
 :help
 %SPHINXBUILD% -M help %SOURCEDIR% %BUILDDIR% %SPHINXOPTS% %O%
 
+:pdf
```
Comment (RobPasMue) [inline doc/make.bat:41]: We should try if the PDF build in Windows works before merging.. just to check

**Review by RobPasMue — CHANGES_REQUESTED** [review]
Thanks @Revathyvenugopal162 for implementing this! Please review my comments and ask for a review again once implemented.

File: doc/make.bat (hunk 2 above)
Comment (Revathyvenugopal162) [reply doc/make.bat:41]: @RobPasMue I checked and in my system it was working.

**Review by RobPasMue — APPROVED** [review]
LGTM!

**Review by RobPasMue — CHANGES_REQUESTED** [review]
We should also include the PDF in the Release process. Please add ./**/*.pdf to the Release section

**Review by RobPasMue — APPROVED** [review]
Now yes! LGTM =)

**Pull request merged**